vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -42,15 +42,29 @@ void AccessibilityInfo::setAccessibilityFocus(double /*reactTag*/) noexcept {
 void AccessibilityInfo::announceForAccessibility(std::string announcement) noexcept {
   m_context.UIDispatcher().Post([context = m_context, announcement = std::move(announcement)] {
     // Windows requires a specific element to announce from. Unfortunately the react-native API does not provide a tag
-    // So we need to add a temporary control to raise the notification event from.
+    // So we need to find something to raise the notification event from.
+    xaml::UIElement element{nullptr};
 
-    auto textBlock = xaml::Controls::TextBlock();
+    if (react::uwp::Is19H1OrHigher()) {
```

Comment:
I think the check you want here is something like `if (react::uwp::IsXamlIsland())`

Reply:
Got it! Didn't realize that check was possible.